astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -446,3 +448,40 @@ def test_time_wcs(time_spectral_wcs_2d):
     # with a time axis.
 
     plt.subplot(projection=time_spectral_wcs_2d)
+
+
+@ignore_matplotlibrc
+def test_simplify_labels_usetex(tmpdir):
+    """Regression test for https://github.com/astropy/astropy/issues/8004."""
+
+    if MATPLOTLIB_LT_22 or not matplotlib.checkdep_usetex(True):
```

Comment:
Turns out the failing version was 2.1.2, so technically it is `>=2.1`. That is why it wasn't working.

Reply:
Thanks! I squashed that fix into my commit. I hope that's OK with you.

Reply:
Yeah, no worries!